### DISCUSSION TITLE
Postgresql: Cannot use multiple schema`s

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#8830] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

I cannot use multiple schema`s for directus.
What i expect is:
 - I give an array of schema`s to DB_SEARCH_PATH
 - Directus uses the first element for its tables
 - All collections are accessible within Directus UI/API

### To Reproduce

Create multiple schema`s inside a PostgresQL database.
Add those to the .env file (for example: DB_SEARCH_PATH=array:schema1,schema2

Directus will throw the following error with every request:
![image](https://user-images.githubusercontent.com/42644889/137537273-5c8f3afd-fe72-483c-905b-e3eb72056e3c.png)

### What version of Directus are you using?

RC96

### What version of Node.js are you using?

Docker

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

Docker

**Comment by joselcvarela** [comment]
Hello @Attacler,
Closing as is duplicate.
If you pretend to use same schema different than 'public', check this: https://github.com/directus/directus/discussions/3164
Or, if you are talking about multiple schemas at once, check this: https://github.com/directus/directus/discussions/8410

**Comment by joselcvarela** [comment]
@rijkvanzanten should I lock this conversation in order to have ideas in just one place?
